kroxylicious-operator/src/test/resources/KafkaProxy-example.yaml

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+kind: KafkaProxy
+apiVersion: kroxylicious.io/v1alpha1
+metadata:
+  name: my-example-proxy
+spec:
+  bootstrapServers: my-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092
```

Comment:
100% fine with this for the first iteration. Though it would be interesting to think about a declarative version of this. e.g. 
```yaml
 spec:
  bootstrapServers: 
      Kind: kafka
      name: YYY
      labels: 
         - app: foo
         - access: public
      namespace: centralCluster
      listener: ??
```
      
Obviously thats a dependency on having Strimzi to manage the Kafka resources on the same cluster so we would always need to support both structures but worth kicking around I feel.

Reply:
Yeah, that would be possible. But it's a step that Strimzi has never taken for their own resources (e.g. `KafkaConnect` accepts bootstrap servers, not a Kafka cluster). It's only mildly more convenient, but supporting `bootstrapServers` means you can support non-Strimzi-based clusters, which some people appreciate.